fixed some mistakes (#167)
OpenLiberty/guide-liberty-deep-dive: README.adoc

----
include::finish/module-health-checks/src/main/java/io/openliberty/deepdive/rest/health/ReadinessCheck.java[]
----

> The [hotspot=Readiness file=2]`@Readiness` annotation indicates that this class is a readiness health check procedure. Navigate to the http://localhost:9080/health/ready URL to check the status of the readiness health check. This tests the connection to the PostgreSQL container that was created earlier in the guide. If the connection is refused, a status of `DOWN` is returned.

Reviewer: ```suggestion
The [hotspot=Readiness file=2]`@Readiness` annotation indicates that this class is a readiness health check procedure. Navigate to the http://localhost:9080/health/ready URL to check the status of the readiness health check. This readiness check tests the connection to the PostgreSQL container that was created earlier in the guide. If the connection is refused, a status of `DOWN` is returned.
```

kubectl port-forward svc/inventory-deployment 9443
```

> The `port-forward` command should hold up the command-line session.

Reviewer: what does "hold up" mean in this context. Also, we should avoid using "should" to refer to a function as it implies uncertainty.

Author: - the word "should" can be removed
- "hold up" means that the command session cannot be typed in anything more. Any better word?

> The `port-forward` command should hold up the command-line session.

Reviewer: ```suggestion
The `port-forward` command pauses the command-line session until you click Ctrl+C.
```

ifndef::cloud-hosted[]

> The application might take some time to get ready. See the https://localhost:9443/health URL to confirm that the `inventory` microservice is up and running. You can now check out the service at the https://localhost:9443/openapi/ui/[https://localhost:9443/openapi/ui/^] URL. The servers dropdown list shows the `\https://localhost:9443/dev` URL. Or, you can run the following command to access the inventory microservice:

Reviewer: ```suggestion
The application might take some time to get ready. See the https://localhost:9443/health URL to confirm that the `inventory` microservice is up and running. You can now check out the service at the https://localhost:9443/openapi/ui/[https://localhost:9443/openapi/ui/^] URL. The server's dropdown list shows the `\https://localhost:9443/dev` URL. Or, you can run the following command to access the inventory microservice:
```
